content/tutorial/tutorial.md

```diff
@@ -1170,41 +1177,41 @@ We will also replace reading `this.state.history` with `this.state.history.slice
   }
 ```
 
-Finally, we will modify the Game component's `render` method from always rendering the last move to rendering the currently selected move according to `stepNumber`:
+अंत में, हम Game कौम्पोनॅन्ट के `render` मेथड को हमेशा अंतिम चरण प्रदान करने से लेकर वर्तमान में चुने गए कदम को `stepNumber` के अनुसार रेंडर करेंगे।
 
 ```javascript{3}
   render() {
     const history = this.state.history;
     const current = history[this.state.stepNumber];
     const winner = calculateWinner(current.squares);
 
-    // the rest has not changed
+    // और इसके आगे कोई चाल बाकि नहीं है
```

Comment:
कोई चाल बाकि नहीं है is not equivalent to changed

Reply:
> कोई चाल बाकि नहीं है is not equivalent to changed

`और कुछ नहीं बदला है` ?

Reply:
how about: iske ilawa aur kuch nai badala hai
?